src/android/CameraLauncher.java

```diff
@@ -163,7 +167,15 @@ else if ((srcType == PHOTOLIBRARY) || (srcType == SAVEDPHOTOALBUM)) {
             PluginResult r = new PluginResult(PluginResult.Status.NO_RESULT);
             r.setKeepCallback(true);
             callbackContext.sendPluginResult(r);
-            
+
+            return true;
+        } else if (action.equals("checkForSavedResult")) {
+            this.imageUri = Uri.fromFile(createCaptureFile(JPEG));
+            if (savedRequestCode > 0) {
+                onActivityResult(savedRequestCode, savedResultCode, savedIntent);
+            } else {
+                callbackContext.success("no-result");
```

Comment:
Would be much nicer if returned result was false when coerced to a boolean (so that they can do if (result)). So maybe return empty string or null.